### PR TITLE
Bump rails to 6.1.4

### PR DESCRIPTION
**Pull request by Riddlerrr**
task: https://www.pivotaltracker.com/story/show/178775172

### CODE REVIEW

**Comment by Riddlerrr** [comment]
It will work after  getting rid of fortitude. Checked here https://github.com/CoMakery/comakery-server/compare/bump-rails-wo-furtitude?expand=1

**Review by zhinfuhnya — APPROVED** [review]

File: app/models/invite.rb [hunk 1]
```diff
@@ -16,11 +14,6 @@ class Invite < ApplicationRecord
 
   scope :pending, -> { where(accepted: false) }
 
-  # Override ActiveRecord::SecureToken class method (remove after upgrade to rails 6.1)
```
Comment (Alexey1100) [inline app/models/invite.rb:19]: Nice :)

File: app/models/mission.rb [hunk 2]
```diff
@@ -59,6 +55,18 @@ def serialize
...
 
+  def unarchived_projects
+    projects.not_archived
+  end
+
+  def ready_award_types
+    AwardType.where(project: unarchived_projects).public_state
+  end
+
+  def published_awards
```
Comment (Alexey1100) [inline app/models/mission.rb:66]: Why is this moved from a relation into a method?

File: app/models/project.rb [hunk 3]
```diff
@@ -206,6 +203,14 @@ def ready_tasks
     awards.ready.includes(:project, :issuer, :account, :award_type)
   end
 
+  def ready_award_types
+    award_types.public_state
+  end
+
+  def published_awards
```
Comment (Alexey1100) [inline app/models/project.rb:210]: And here :)

**Review by Alexey1100 — CHANGES_REQUESTED** [review]

**Review by Alexey1100 — APPROVED** [review]

**Review by aquabu — APPROVED** [review]

**Pull request merged**